src/network_analysis.jl

```diff
@@ -779,12 +780,12 @@ function iscomplexbalanced(rs::ReactionSystem, parametermap::Dict)
     end
 end
 
-function iscomplexbalanced(rs::ReactionSystem, parametermap::Vector{Pair{Symbol, Float64}})
+function iscomplexbalanced(rs::ReactionSystem, parametermap::Vector{<:Pair})
```

Comment:
These functions (and similar for `ratematrix`) could not accept inputs of the form of Symbolics. I made the minimal change so that this (and the tests added below) now should work. Is this OK with you @vyudu? We might want to make them a bit more stringent so that not symbol/symbolic inputs do not work, in this case I tried to make changes minimal though (and on parity with how the `Dict` input was handled.

Reply:
This makes sense also - thanks for adding these.